src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,39 +54,43 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        try {
```

Comment:
maybe make some checks for input params

Reply:
let's keep it as it is.
try-catch is good for this checking